lib/scientist/experiment.rb

```diff
@@ -269,4 +269,16 @@ def try(name = nil, &block)
   def use(&block)
     try "control", &block
   end
+
+  def raise_on_mismatches=(bool)
+    @raise_on_mismatches = bool
+  end
+
+  def raise_on_mismatches?
+    if @raise_on_mismatches.nil?
+      self.class.raise_on_mismatches?
```

Comment:
Would we want to keep the class-level attribute? Not saying we shouldn't, as there are advantages, just wanted to underline the decision.